<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OCPP_CS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="Charger1">
       <attribute name="title">
        <string>Cargador 1</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>30</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QLabel" name="label_estado_general">
            <property name="font">
             <font>
              <pointsize>16</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LayoutDirection::LeftToRight</enum>
            </property>
            <property name="text">
             <string>ESTADO DEL CARGADOR 1: No conectado</string>
            </property>
            <property name="textFormat">
             <enum>Qt::TextFormat::MarkdownText</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_vendor">
            <property name="text">
             <string>chargePointVendor: Desconocido</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_model">
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>chargePointModel: Desconocido</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>23</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="frame_connector1">
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,3,2">
           <item>
            <widget class="QLabel" name="label_img_conector1">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>CONNECTOR 1</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_idTag1">
               <property name="text">
                <string>idTag: (cargador sin 
ninguna transacción</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_transactionId1">
               <property name="text">
                <string>transactionId: (cargador sin 
ninguna transacción</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QComboBox" name="operaciones1">
               <item>
                <property name="text">
                 <string>ChangeAvailability</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>ClearCache</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>DataTransfer</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>GetConfiguration</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>RemoteStartTransaction</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>RemoteStopTransaction</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Reset</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>UnlockConnector</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="mostrar_operacion1">
               <property name="text">
                <string>Mostrar Operación</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Policy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>8</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QFrame" name="frame_connector2">
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Raised</enum>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,3,2">
           <item>
            <widget class="QLabel" name="label_img_conector2">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_5">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="font">
                <font>
                 <bold>true</bold>
                </font>
               </property>
               <property name="text">
                <string>CONNECTOR 2</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_idTag2">
               <property name="text">
                <string>idTag: (cargador sin 
ninguna transacción</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_transactionId2">
               <property name="text">
                <string>transactionId: (cargador sin 
ninguna transacción</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_6">
             <item>
              <widget class="QComboBox" name="operaciones2">
               <item>
                <property name="text">
                 <string>ChangeAvailability</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>ClearCache</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>DataTransfer</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>GetConfiguration</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>RemoteStartTransaction</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>RemoteStopTransaction</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Reset</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>UnlockConnector</string>
                </property>
               </item>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="mostrar_operacion2">
               <property name="text">
                <string>Mostrar Operación</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="Charger2">
       <attribute name="title">
        <string>Cargador 2</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="Charger3">
       <attribute name="title">
        <string>Cargador 3</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="Charger4">
       <attribute name="title">
        <string>Cargador 4</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>746</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuEstadisticas">
    <property name="title">
     <string>Estadísticas</string>
    </property>
    <addaction name="actionEstadisticas"/>
   </widget>
   <addaction name="menuEstadisticas"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEstadisticas">
   <property name="text">
    <string>Ver Estadísticas</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
